README.md

```diff
@@ -1,602 +1,361 @@
-routable.
-
-```ruby
-expect(:get => "/widgets/1/edit").not_to be_routable
-```
-
-### `have_http_status`
-
-- Passes if `response` has a matching HTTP status code
-- The following symbolic status codes are allowed:
-  - `Rack::Utils::SYMBOL_TO_STATUS_CODE`
-  - One of the defined `ActionDispatch::TestResponse` aliases:
-    - `:error`
-    - `:missing`
-    - `:redirect`
-    - `:success`
-- Available in controller, feature, and request specs.
-
-In controller and request specs, apply to the `response` object:
-
-```ruby
-expect(response).to have_http_status(201)
-expect(response).not_to have_http_status(:created)
-```
-
-In feature specs, apply to the `page` object:
-
-```ruby
-expect(page).to have_http_status(:success)
-```
+Request specs provide a high-level alternative to controller specs.
+In fact, as of RSpec 3.5, both the Rails and RSpec teams
+[discourage directly testing controllers][]
+in favor of functional tests like request specs.
 
-## `rake` tasks
+When writing them, try to answer the question,
+“For a given HTTP request (verb + path + parameters),
+what HTTP response should the application return?”
 
-Several rake tasks are provided as a convenience for working with RSpec. To run
-the entire spec suite use `rake spec`. To run a subset of specs use the
-associated type task, for example `rake spec:models`.
+[discourage directly testing controllers]: rspec.info/blog/2016/07/rspec-3-5-has-been-released/#rails-support-for-rails-5
 
-A full list of the available rake tasks can be seen by running `rake -T | grep
-spec`.
+## Contributing
 
-### Customizing `rake` tasks
+- [Build details](BUILD_DETAIL.md)
+- [Code of Conduct](CODE_OF_CONDUCT.md)
+- [Detailed contributing guide](CONTRIBUTING.md)
 
-If you want to customize the behavior of `rake spec`, you may [define your own
-task in the `Rakefile` for your
-project](https://www.relishapp.com/rspec/rspec-core/docs/command-line/rake-task).
-However, you must first clear the task that rspec-rails defined:
+Once you’ve cloned the repo and [set up the environment](DEVELOPMENT.md),
+you can run the specs and Cucumber features, or submit a pull request.
 
-```ruby
-task("spec").clear
-```
+## See Also
 
+### RSpec base libraries
 
-## Also see
+* <https://github.com/rspec/rspec>
+* <https://github.com/rspec/rspec-core>
+* <https://github.com/rspec/rspec-expectations>
+* <https://github.com/rspec/rspec-mocks>
 
-* [https://github.com/rspec/rspec](https://github.com/rspec/rspec)
-* [https://github.com/rspec/rspec-core](https://github.com/rspec/rspec-core)
-* [https://github.com/rspec/rspec-expectations](https://github.com/rspec/rspec-expectations)
-* [https://github.com/rspec/rspec-mocks](https://github.com/rspec/rspec-mocks)
+### Recommended third-party extensions
 
-## Feature Requests & Bugs
+* [FactoryBot](https://github.com/thoughtbot/factory_bot)
+* [Capybara](https://github.com/jnicklas/capybara)  
+  (Included by default in Rails 5.1+.
```

Comment:
It should be on the same line as 356 no?

Reply:
The newline serves two purposes:

1. to keep each line of text under 80 chars, and
2. to add a `<br>` between the Capybara link and the parenthetical note

That is, this markdown:

```markdown
* foo<space><space>
  bar
```

compiles to this HTML:

```html
<li>
  foo
  <br>
  bar
</li>
```

Originally, this was my preferred formatting, since the only other way to make a multi-line list item is with `\n\n`, which compiles to a `<p>` and results in a lot of vertical padding. But since now it looks like trailing spaces break the CI, I'm happy to just keep it all on one line instead.